generators/bundler-webpack/templates/webpack.config.js

```diff
@@ -20,50 +18,36 @@ module.exports = {
   module: {
     rules: [
       {
-        test: /\.{js|jsx}$/,
+        test: /\.jsx?$/,
         exclude: /node_modules/,
         use: {
           loader: 'babel-loader',
-          options: {
-            presets: ['env', ['es2015', { modules: false }], 'react'],
-          },
         },
       },
       {
         test: /\.scss$/,
```

Comment:
Ran into a couple cases lately where I needed a plain CSS loader, usually for dealing with third party libs:

``` javascript
      {
        test: /\.css$/,
        use: ExtractTextPlugin.extract({
          fallback: 'style-loader',
          use: ['css-loader']
        }),
      },
```